src/components/ChannelCard.tsx

```diff
@@ -1,39 +1,44 @@
 import React from 'react'
 
 import { useChannel } from '@/api/hooks'
-import { useChannelVideoCount } from '@/api/hooks/channel'
-import { absoluteRoutes } from '@/config/routes'
+import { useHandleFollowChannel } from '@/hooks'
 import { AssetType, useAsset } from '@/providers'
 import { ChannelCardBase } from '@/shared/components'
 
-type ChannelCardProps = {
+export type ChannelVariant = 'primary' | 'secondary'
+
+export type ChannelCardProps = {
   id?: string
+  rankingNumber?: number
+  isLoading?: boolean
+  variant?: ChannelVariant
```

Comment:
What does this variant refer to?

Reply:
this is `primary` variant
![image](https://user-images.githubusercontent.com/51168865/129051565-bd55a37e-edb8-402c-aca7-99956d3c4c76.png)
and this is `secondary` 
![image](https://user-images.githubusercontent.com/51168865/129051675-3d620809-ebdf-4112-b2af-6ff822e04478.png)